templates.rst

```diff
@@ -0,0 +1,1061 @@
+                controller="App\Controller\BlogController::show"/>
+        </routes>
+
+    .. code-block:: php
+
+        // config/routes.php
+        use App\Controller\BlogController;
+        use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+        return function (RoutingConfigurator $routes) {
+            $routes->add('blog_index', '/')
+                ->controller([BlogController::class, 'index'])
+            ;
+
+            $routes->add('blog_post', '/articles/{slug}')
+                ->controller([BlogController::class, 'show'])
+            ;
+        };
+
+Use the ``path()`` Twig function to link to these pages and pass the route name
+as the first argument and the route parameters as the optional second argument:
+
+.. code-block:: html+twig
+
+    <a href="{{ path('blog_index') }}">Homepage</a>
+
+    {# ... #}
+
+    {% for post in blog_posts %}
+        <h1>
+            <a href="{{ path('blog_post', {slug: post.slug}) }}">{{ post.title }}</a>
+        </h1>
+
+        <p>{{ post.excerpt }}</p>
+    {% endfor %}
+
+The ``path()`` function generates relative URLs. If you need to generate
+absolute URLs (for example when rendering templates for emails or RSS feeds),
+use the ``url()`` function, which takes the same arguments as ``path()``
+(e.g. ``<a href="{{ url('blog_index') }}"> ... </a>``).
+
+.. _templates-link-to-assets:
+
+Linking to CSS, JavaScript and Image Assets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Templates usually link to images, JavaScript, stylesheets and other assets.
+Instead of writing the asset URLs by hand, use the ``asset()`` function to
+generate the URLs.
+
+To use this function, install the *asset* package:
+
+.. code-block:: terminal
+
+    $ composer require symfony/asset
+
+You can now use the ``asset()`` function:
+
+.. code-block:: html+twig
+
+    <img src="{{ asset('images/logo.png') }}" alt="Symfony!"/>
+
+    <link href="{{ asset('css/blog.css') }}" rel="stylesheet"/>
+
+    <script src="{{ asset('bundles/acme/js/loader.js') }}"></script>
+
+The ``asset()`` function's main purpose is to make your application more portable.
+If your application lives at the root of your host (e.g. ``https://example.com``),
+then the rendered paths should be ``/images/logo.png``. But if your application
+lives in a subdirectory (e.g. ``https://example.com/my_app``), each asset path
+should render with the subdirectory (e.g. ``/my_app/images/logo.png``). The
+``asset()`` function takes care of this by determining how your application is
+being used and generating the correct paths accordingly.
+
+.. tip::
+
+    The ``asset()`` function supports various cache busting techniques via the
+    :ref:`version <reference-framework-assets-version>`,
+    :ref:`version_format <reference-assets-version-format>`, and
+    :ref:`json_manifest_path <reference-assets-json-manifest-path>` configuration options.
+
+If you need absolute URLs for assets, use the ``absolute_url()`` Twig function
```

Comment:
Super minor, but `absolute_url()` is also covered (briefly) in `reference/twig_reference.rst`. Probably a little bit of duplication is ok...